evalml/pipelines/components/transformers/encoders/onehot_encoder.py

```diff
@@ -91,7 +89,9 @@ def __init__(
     @staticmethod
     def _get_cat_cols(X):
         """Get names of categorical columns in the input DataFrame."""
-        return list(X.ww.select(include=["category"]).columns)
+        return _put_into_original_order(
```

Comment:
I'm putting in the same order as they're found in the dataframe to keep the behavior the same.

Reply:
I was surprised by this! Yea because `.columns` on the schema returns a dictionary keyed by column name. So when you convert that to a list the column order could be different from time to time.